base/applications/rapps/lang/es-ES.rc

```diff
@@ -214,9 +214,9 @@ BEGIN
     IDS_SELECTEDFORINST "Seleccionados para instalar"
     IDS_MISMATCH_CERT_INFO "El certificado que usa es desconocido:\nSujeto: %s\nEmisor: %s\n¿Quiere continuar a pesar de ello?"
     IDS_UNABLE_PATH "Formato de ruta incorrecto."
-    IDS_APP_DISPLAY_DETAILS "details"
-    IDS_APP_DISPLAY_LIST "list"
-    IDS_APP_DISPLAY_TILE "tile"
+    IDS_APP_DISPLAY_DETAILS "Detalles"
```

Comment:
Currently github tells there's a conflict here with master, because I've capitalized these three strings. You'll need to fix this small conflict in your branch, after rebasing, by overwriting the modifications in this file from master, by your changes.